package.json

```diff
@@ -74,9 +79,10 @@
 		"ts-loader": "^9.2.8",
 		"ts-node": "^10.4.0",
 		"typescript": "^4.5.2",
+		"webpack": "^5.70.0",
+		"@chainsafe/geth-dev-assistant": "0.1.11",
```

Comment:
Please make sure that there is no critical and no high vulnerabilities. Currently it have: **(3 high, 1 critical)**
It uses: 
```
minimist  <1.2.6 with
Severity: critical vulnerability 
```

Reply:
@spacesailor24 or @nazarhussain could one of you update and publish @chainsafe/geth-dev-assistant for removing this security issue,